### DISCUSSION TITLE
Fine tune ANCE

### DISCUSSION THREAD

**Discussion by noem-1998** [castorini/pyserini#1417] [Q&A]
Hi, is there a code example of how to fine tune a model such as ANCE on some dataset like one of the BEIR benchmark?

**Comment by MXueguang** [comment]
Hi @noem-1998 
You can use Tevatron to finetune models.
here is an example to finetune BERT with scifact training data
https://github.com/texttron/tevatron/blob/main/examples/example_scifact.md
